### PR TITLE
NIT-580 add nd interface config file to iaps ami

### PR DESCRIPTION
**Pull request by pete-j-g**

### CODE REVIEW

File: teams/delius-iaps/components/files/IAPSCMSIF.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      <DEFPROGTEAM CODE="ASP">ASP nDelius Awaiting Allocation</DEFPROGTEAM>
+      <DEFPROGTEAM CODE="HFS">HFS Referral Clearing</DEFPROGTEAM>
+    </DEFPROGTEAMS>
+    <CONTACTCLASSES>
+      <CONTACTCLASS CODE="Pre-programme">Session</CONTACTCLASS>
+      <CONTACTCLASS CODE="Post-programme">Session</CONTACTCLASS>
+      <CONTACTCLASS CODE="DEFAULT">Log</CONTACTCLASS>
+    </CONTACTCLASSES>
+  </CODELOOKUPS>
+</DLLDEF>
```
Comment (kylehodgetts) [inline teams/delius-iaps/components/files/IAPSCMSIF.xml:90]:
```suggestion
</DLLDEF>

```

File: teams/delius-iaps/components/files/IAPSCMSIF.xml [hunk 2]
```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```
Comment (kylehodgetts) [inline teams/delius-iaps/components/files/IAPSCMSIF.xml:1]: Looks good but could you share where you got this file from or the knowledge base which helped you put this together?

Comment (pete-j-g) [reply teams/delius-iaps/components/files/IAPSCMSIF.xml:1]: This was taken from from the old stage environment. These files have never been maintained by us, or the team previously in charge of the infrastructure. To our knowledge the files would be manually edited by the service team at i2n/necsws.

**Review by kylehodgetts — APPROVED** [review]

File: teams/delius-iaps/components/files/IAPSCMSIF.xml (hunk 2 above)
Comment (pete-j-g) [reply teams/delius-iaps/components/files/IAPSCMSIF.xml:1]: To add to this, the application comes bundled with a sample config file, which contains fewer DEFPROGTEAM entries.

**Pull request merged**